pkg/controller/tf/controller.go

```diff
@@ -363,6 +363,12 @@ func (r *Reconciler) sync(ctx context.Context, krmResource *krmtotf.Resource) (r
 	if err != nil {
 		return false, r.HandleUpdateFailed(ctx, &krmResource.Resource, fmt.Errorf("error calculating diff: %w", err))
 	}
+	if krmResource.Kind == "ContainerCluster" && diff != nil {
+		fmt.Println("maqiuyu... containercluster...")
```

Comment:
remove ?

Reply:
Good catch! Will address it after #4713 is merged (as I'll need to rebase anyways).

Reply:
Done!